### ISSUE TITLE
Calendar: mask does not appear, only when using useEffect

### ISSUE THREAD

**Issue by B-Schmitz** [primefaces/primereact#4698]
My component return:

```ts
return (
<Calendar
            mask={"99/99/9999 99:99"}
            value={value}
            onChange={onChange}
            showTime
            dateFormat={"dd/mm/yy"}
            placeholder={"dd/mm/yyyy hh:mm"}
            hideOnDateTimeSelect
            showButtonBar
            showIcon
            showOnFocus={false}
        />
     );
```

That way it doesn't work, but when I using useEffect the mask always appears correctly!

```ts
const [mask, setMask] = useState("");

   useEffect(() => {
        setMask("99/99/9999 99:99");
    }, []);

return (
        <Calendar
            mask={mask}
            value={new Date(p.valor)}
            onChange={onChange}
            showTime
            dateFormat={"dd/mm/yy"}
            placeholder={"dd/mm/yyyy hh:mm"}
            hideOnDateTimeSelect
            showButtonBar
            showIcon
            showOnFocus={false}
        />
    );

```

### PrimeReact version

9.6.0

### React version

18.x

### Language

TypeScript

### Build / Runtime

Next.js

**Comment by melloware** [comment]
Yep I already fixed this for 9.6.1 just need to find the ticket.

**Comment by melloware** [comment]
Duplicate of #4579
